sdk/canton/daml-common-staging/daml-errors/src/main/scala/com/daml/error/SerializableErrorComponents.scala

```diff
@@ -215,7 +215,7 @@ private[error] final case class NonSecuritySensitiveErrorCodeComponents(
 private[error] object NonSecuritySensitiveErrorCodeComponents {
 
   /** The maximum size (in characters) of the self-service error description, truncated for transport as part of a Status */
-  val MaxCauseLogLength = 1024
```

Comment:
I recall there was some discussion around changing this value for canton. Could I just check what the outcome of those discussions were pleased? Just being cautious as I'm aware that changing this value may impact production debugging efforts.

Reply:
I raised it to 1024 in a previous PR, and we decided that would cause issues - this is reverting it back to 512 along with some test changes to cope with that

Reply:
Ah, got you. Thanks for pointing that out.